templates/default/tests/markdownFeatures.spec.js

```diff
@@ -0,0 +1,136 @@
+const { test, expect } = require('@playwright/test');
+const { MarkdownPage } = require('./markdown-page');
+
+test.describe('wiki links', () => {
+  test.beforeEach(async ({ page }) => {
+    const Page = new MarkdownPage(page);
+    await Page.goto('/test/fixtures/markdownFeatures');
+    await Page.getData(page);
+  });
+
+  test('parses a wiki link', async ({ page }) => {
+    const link = page.locator('#wikiLink > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/docs/roadmap');
+  });
+
+  test('parses a wiki link with custom divider', async ({ page }) => {
+    const link = page.locator('#wikiLink-custom > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/docs/roadmap');
+  });
+
+  test('parses a wiki link with header', async ({ page }) => {
+    const link = page.locator('#wikiLink-heading > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/docs/roadmap#planned-features');
+  });
+
+  test('parses a wiki link with header and custom divider', async ({ page }) => {
+    const link = page.locator('#wikiLink-heading-custom > p > a');
+    await expect(link).toHaveClass('internal');
+    await link.click();
+    await expect(page).toHaveURL('/docs/roadmap#planned-features');
+  });
+
+  test('link to image file', async ({ page }) => {
+    const link = page.locator('#wikiLink-image > p > img');
+    await expect(link).toHaveClass('internal');
+    await expect(link).toHaveAttribute('src', '/assets/images/park.png');
```

Comment:
This will also break if we remove the `park.png` image when we don't need it anymore. Let's copy this image to where the markdown fixtures are (maybe in some `/assets` subfolder for now, as I'm not sure what is the best practice here)

Reply:
I added a testimages subfolder to images for now, but we can change the location if there is better practice for this.